src/Features/Auth/Login/index.js

```diff
@@ -10,7 +10,6 @@ function LoginForm() {
   const dispatch = useDispatch();
 
   const loginClickHandler = () => {
-    console.log('asdsad');
     dispatch(login({ email: loginForm.email, password: loginForm.password }));
```

Comment:
As login feature has been integrated in development branch so try to use the input entered by the user